### PR TITLE
Adding a logo on Firefly boards

### PR DESCRIPTION
**Pull request by S199pWa1k9r**

### CODE REVIEW

File: edk2-rockchip/Platform/Firefly/ROC-RK3566-PC/ROC-RK3566-PC.dsc [hunk 1]
```diff
@@ -470,8 +470,8 @@
 
   gEfiMdeModulePkgTokenSpaceGuid.PcdVideoHorizontalResolution|0
   gEfiMdeModulePkgTokenSpaceGuid.PcdVideoVerticalResolution|0
-  gEfiMdeModulePkgTokenSpaceGuid.PcdSetupVideoHorizontalResolution|1280
-  gEfiMdeModulePkgTokenSpaceGuid.PcdSetupVideoVerticalResolution|720
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSetupVideoHorizontalResolution|1920
```
Comment (jaredmcneill) [inline edk2-rockchip/Platform/Firefly/ROC-RK3566-PC/ROC-RK3566-PC.dsc:473]: Unrelated change

File: edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c [hunk 2]
```diff
@@ -1000,7 +1000,7 @@ PhyMemArrayInfoUpdateSmbiosType16 (
  //  - Type 17 VolatileSize in Bytes
  //
 
-  mMemDevInfoType17.Size = PcdGet64(PcdSystemMemorySize) / (1024 * 1024);;
+  mMemDevInfoType17.Size = mMemorySize / (1024 * 1024);
```
Comment (jaredmcneill) [inline edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c:1003]: Oops good catch. Can you submit this in a separate change?

File: edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/ROC-RK3568-PC.dsc [hunk 3]
```diff
@@ -388,7 +388,7 @@
 
   # Smallest possible memory size
   gArmTokenSpaceGuid.PcdSystemMemoryBase|0
-  gArmTokenSpaceGuid.PcdSystemMemorySize|0x80000000
+  gArmTokenSpaceGuid.PcdSystemMemorySize|0x40000000
```
Comment (jaredmcneill) [inline edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/ROC-RK3568-PC.dsc:391]: Separate change

File: edk2-rockchip/Platform/Firefly/Drivers/LogoDxe/License.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+The source code in this directory is licensed under the standard EDK2 terms.
+
```
Comment (jaredmcneill) [inline edk2-rockchip/Platform/Firefly/Drivers/LogoDxe/License.txt:2]: Please add a note about the license terms for Logo.bmp

File: edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c (hunk 2 above)
Comment (S199pWa1k9r) [reply edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c:1003]: Yes, I will separate everything into separate pull requests

File: edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/ROC-RK3568-PC.dsc (hunk 3 above)
Comment (S199pWa1k9r) [reply edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/ROC-RK3568-PC.dsc:391]: OK

File: edk2-rockchip/Platform/Firefly/Drivers/LogoDxe/License.txt (hunk 4 above)
Comment (S199pWa1k9r) [reply edk2-rockchip/Platform/Firefly/Drivers/LogoDxe/License.txt:2]: Due to the big time difference this document can take a long time, 
but i already sent a request to Firefly

Comment (spikerguy) [reply edk2-rockchip/Platform/Firefly/Drivers/LogoDxe/License.txt:2]: > Please add a note about the license terms for Logo.bmp

They did agree to allow use of the logo. I am waiting for their license document so we can attach it here if needed.

@jaredmcneill I tried contacting you on twitter @spikerguy and also on your email but no reply. It is regarding Device sample for you.

Comment (andreamtp) [reply edk2-rockchip/Platform/Firefly/Drivers/LogoDxe/License.txt:2]: @S199pWa1k9r @spikerguy have you received any feedback from Firefly in this regard?
